python/ray/experimental/workflow/storage/__init__.py

```diff
@@ -11,21 +15,56 @@ def create_storage(storage_url: str) -> Storage:
 
     Args:
         storage_url: A URL indicates the storage type and root path.
+        Currently only two types of storages are supported: local fs and s3
+        For local fs, a path is needed, i.e.:
+           file:///local_path
+
```

Comment:
Should omit file:///

Reply:
Since the input is `storage_url`, I guess we should make it a valid URI. I don't mind making the default one file, in case the scheme is not specified.